examples/TemplateGenerationIterative.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>We should avoid the use of <code>filter</code>  for a variable name because it conflicts with the built-in function <code>filter</code>.  We can use <code>filt</code>  or <code>filter_</code>  or preferably something more specific, e.g. <code>paster</code>  in this case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/HASI/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>